loki-build-image/Dockerfile

```diff
@@ -99,11 +104,12 @@ COPY --from=jsonnet /go/bin/jsonnet /usr/bin/jsonnet
 # It's possible this can be revisited in newer versions of Go if the behavior around GOPATH vs GO111MODULES changes
 RUN GO111MODULE=on go install github.com/golang/protobuf/protoc-gen-go@v1.3.1
 RUN GO111MODULE=on go install github.com/gogo/protobuf/protoc-gen-gogoslick@v1.3.0
-    # Due to the lack of a proper release tag, we use the commit hash of
-    # https://github.com/golang/tools/releases v0.1.7
+# Due to the lack of a proper release tag, we use the commit hash of
+# https://github.com/golang/tools/releases v0.1.7
 RUN GO111MODULE=on go install golang.org/x/tools/cmd/goyacc@58d531046acdc757f177387bc1725bfa79895d69
 RUN GO111MODULE=on go install github.com/mitchellh/gox@9f71238 && rm -rf /go/pkg /go/src
 ENV GOCACHE=/go/cache
+ENV GOTEST="gotestsum --format testname --"
```

Comment:
why this `format` exactly?

Reply:
It prints the testname with PASS/FAIL and the time taken. The other formats print the package name only.